README.md

```diff
@@ -134,6 +134,13 @@ of the most common ones with non default values. For more info take a look at th
   Token used to verify the device identity with Google.
   See more in https://developers.google.com/identity/protocols/OAuth2
 
+#### File Storage System (Used on Kernel, ODK and UI Modules)
```

Comment:
I thought that this was only needed for Kernel and ODK to save the media content (attachments in kernel and xForm media files in ODK) :thinking:

Reply:
https://docs.djangoproject.com/en/2.0/topics/files/#the-built-in-filesystem-storage-class

Reply:
UI was mainly included for the static files option

Reply:
With the same argument, what happens with `couchdb-sync`?

Reply:
As long as `common` is a dependency on any module. s3 storage feature will be available with the same arguments

Reply:
But it's not included in the PR :grimacing:

Reply:
Yea. Changes due to  your 1st comment will address that 😁